packages/dashboard-backend/src/__tests__/app.spec.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2018-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { FastifyInstance } from 'fastify';
+import { setup, teardown } from '../helpers/tests/appBuilder';
+
+describe('App', () => {
```

Comment:
I’m not sure I get what this test does.

Reply:
It tests that the app can start. I'll update the test to make that clear.